contents/docs/getting-started/_snippets/user-properties-how-to-set.mdx

```diff
@@ -1,10 +1,10 @@
-The recommended way to set person properties in person profiles is to include them when capturing an event:
+The recommended way to set properties in person profiles is to use `$set`:
```

Comment:
I slightly messed with this in my merge (because I didn't realize we were recommending sending a `$set` event too). I think this wording captures what you're going for. 
```suggestion
The recommended way to set properties in person profiles is to send a `$set` event with a `$set` property:
```